service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpSend.java

```diff
@@ -157,13 +157,15 @@ public AmqpSend(
     /**
      * Create AmqpsSendHandler and store it in a member variable
      */
+    @SuppressWarnings("EmptyMethod")
```

Comment:
are open and close needed?

Reply:
..I guess removing them would be a breaking change.

Reply:
Correct